microlensing_photometry/astrometry/wcs.py

```diff
@@ -0,0 +1,93 @@
+from skimage.registration import phase_cross_correlation
+import numpy as np
+from astropy.coordinates import SkyCoord
+import scipy.spatial as sspa
+from skimage.measure import ransac
+from skimage import transform as tf
+from astropy.wcs import WCS,utils
+
+import microlensing_photometry.logistics.GaiaTools.GaiaCatalog as GC
+
+def find_images_shifts(reference,image,image_fraction =0.25, upsample_factor=1):
+    """
+    Estimate the shifts between two images. Generally a good idea to do a fraction in the middle,
+    where the effect of rotation is minimal.
+
+    Parameters
+    ----------
+    reference : an image acting as the reference
```

Comment:
The parameter types should be specified in each case - for example, are the images being passed here as arrays?  File paths?